web/src/components/TestSpecForm/AssertionCheckList.tsx

```diff
@@ -37,7 +34,7 @@ const AssertionCheckList: React.FC<IProps> = ({form, fields, add, remove, attrib
               attributeList={attributeList}
               name={name}
               index={index}
-              assertionList={assertionList}
```

Comment:
Why don't you like the list naming for arrays? 😄

Reply:
Personal preference. I don't think you need to explain with the name the type of variable. However I'm ok with that.